aioresponses/core.py

```diff
@@ -234,6 +278,8 @@ def wrapped(*args, **kwargs):
     def clear(self):
         self._responses.clear()
         self._matches.clear()
+        self._call_args = None
```

Comment:
the name of that variable suggest that this is call arguments, but type annotation is `RequestMatch` :)